zerver/lib/test_helpers.py

```diff
@@ -228,12 +228,12 @@ class AuthedTestCase(TestCase):
     # Helper because self.client.patch annoying requires you to urlencode
 
     def client_patch(self, url, info={}, **kwargs):
-        # type: (str, Any, Dict[str, Any]) -> HttpResponse
+        # type: (str, Any, **Any) -> HttpResponse
         info = urllib.parse.urlencode(info)
         return self.client.patch(url, info, **kwargs)
 
     def client_put(self, url, info={}, **kwargs):
-        # type: (str, Any, Dict[str, Any]) -> HttpResponse
+        # type: (str, Any, **Any) -> HttpResponse
```

Comment:
should still be Dict[str, Any] for `info` -- but I notice when doing this you get an error because on the next line `info`'s type changed.  I fixed this by renaming `info` to `encoded` where it's assigned on the next line (and on the line after) for these three, and will merge.